### PR TITLE
Basic Inventory UI

### PR DESCRIPTION
**Pull request by matekdev**
Good enough to be styled at some point

https://user-images.githubusercontent.com/32153021/218911656-be7fa923-8f9a-4739-94d1-ecf386e382bd.mp4

### CODE REVIEW

File: code/UI/Inventory/InventoryPanel.razor [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <div class="items">
+        @foreach(var weapon in player.Inventory.Weapons)
+        {
+            var isAvailable = weapon.Charges > 0;
+            <InventoryItem
+             Weapon=@weapon
+             IsAvailable=@isAvailable
+             @onmouseover=@(() => _hoveredWeapon = weapon)
+             @onmouseout=@(() => _hoveredWeapon = null)
+             @onclick=@(() => { if (isAvailable) player.Inventory.SetActiveWeapon(weapon); })
```
Comment (matekdev) [inline code/UI/Inventory/InventoryPanel.razor:21]: SetActiveWeapon needs to be changed, it doesn't work when called from client. I'd rather do it in another PR.

1. Either we change the weapon on server (i.e make cmd call)
2. Or we setup `[ClientInput] ActiveChildInput` (this would be preferable

Comment (trundler-dev) [reply code/UI/Inventory/InventoryPanel.razor:21]: We can set up client input for that. If you want to have a go at it this PR feel free. This all looks great, btw.
